src/Test.java

```diff
@@ -2,8 +2,8 @@
 public class Test {
 
 	public static void main(String[] args) {
-String name="neeraj";
-if(name=="neeraj")
+String name="neerajag";
```

Comment:
You don't have to declare the datatype again, to change values.
Just do
name="neerajag";
instead of,
String name= "neerajag";